<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenSharing</class>
 <widget class="QWidget" name="ScreenSharing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>136</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" rowspan="2" colspan="3">
    <widget class="QFrame" name="m_pFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QSpinBox" name="m_pWidth"/>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="m_pHeight"/>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="m_pDisplay"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Display id:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>,</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Offset (x,y):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="m_pX"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Size (width,height):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QSpinBox" name="m_pY"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="m_pScreens"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Screen:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>m_pScreens</zorder>
  <zorder>m_pFrame</zorder>
  <zorder>label</zorder>
  <zorder>label_6</zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pScreens</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ScreenSharing</receiver>
   <slot>slotScreenIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotScreenIndexChanged(int)</slot>
 </slots>
</ui>
